draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -429,7 +428,7 @@ For each claim that is an array element and that is to be made selectively discl
    1. The salt value as described in (#disclosures_for_object_properties).
    2. The array element that is to be hidden. This value can be of any type that is allowed in JSON, including numbers, strings, booleans, arrays, and objects.
 
-The Disclosure string is created by JSON-encoding this array and base64url-encoding the byte representation of the resulting string as described in (#disclosures_for_object_properties). The same considerations regarding
+The Disclosure string is created by base64url-encoding the UTF-8 byte sequence of the resulting JSON array's string representation as described in (#disclosures_for_object_properties). The same considerations regarding
```

Comment:
Please don't use "string representation" unless strings are involved.
This is one place where "JSON encoding" would make sense.

Reply:
It is JSON so I'm not sure "JSON encoding" does make sense.

Reply:
The language is hard.

Reply:
There are lots of strings in involved. So to maybe to phrase it differently, what would you call the thing here https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-18.html#section-5.1-9.3 and in many other places after the label `Contents: `?

Reply:
overcome by events